MatrixKit/Utils/MXKSoundPlayer.m

```diff
@@ -74,9 +74,6 @@ - (void)stopPlaying
     {
         [self.audioPlayer stop];
         self.audioPlayer = nil;
-        
-        // Release the audio session to allow resuming of background music app
-        [[AVAudioSession sharedInstance] setActive:NO withOptions:AVAudioSessionSetActiveOptionNotifyOthersOnDeactivation error:nil];
```

Comment:
Can we keep this code in the case where we are not running CallKit?
My intention is to be able to reuse this MXKSoundPlayer (for local notifications sounds, for example).

stopPlaying could have an additional bool parameter like deactiveAudioSession:.